gekitchen/erd/converters/primitives.py

```diff
@@ -0,0 +1,135 @@
+__all__ = (
+    "ErdIntConverter",
+    "ErdSignedByteConverter",
+    "ErdBytesConverter",
+    "ErdBoolConverter",
+    "ErdStringConverter",   
+    "ErdTimeSpanConverter",
+    "erd_decode_int",
+    "erd_encode_int",
+    "erd_decode_signed_byte",
+    "erd_encode_signed_byte",
+    "erd_decode_bytes",
+    "erd_encode_bytes",
+    "erd_decode_bool",
+    "erd_encode_bool",
+    "erd_decode_string",
+    "erd_encode_string",
+    "erd_decode_timespan",
+    "erd_encode_timespan" 
+)
+
+import logging
+from datetime import timedelta
+from typing import Optional
+
+from .abstract import ErdValueConverter
+
+_LOGGER = logging.getLogger(__name__)
+
+class ErdIntConverter(ErdValueConverter[int]):
```

Comment:
These should probably all be decorated `@staticmethod` since they don't use any member variables